### ISSUE TITLE
unnecessary_chaining, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2476]
something wrong when there are comments,

```abap
    DATA: integer    TYPE i,
          decfloat16 TYPE decfloat16, "sdfsdfsd
          integer1   TYPE int1.
```
